docs/FAQ.md

```diff
@@ -66,6 +66,65 @@ One session of Zulip Terminal represents a connection of one user to one Zulip s
+
+Composing messages could be a tiring task when you have to type all things by yourself. Autocompletes come to rescue here! ZT currently supports some basic autocompletes, for different compose boxes.
+
+### Hotkeys triggering autocomplete
+
+In most places, hitting `tab` allows toggling autocompletes. In ZT, however, `ctrl f` and `ctrl r` are used for toggling purpose:
+* `ctrl f` allows switching between autocomplete suggestions in a forward manner (see the GIF below).
+* `ctrl r` does the same job but in reverse manner.
+
+### Understanding the working
+
+Below are some basic steps to work with autocompletes in ZT (assuming that the user is composing a message):
+1. Type a prefix designated for an autocomplete (e.g., `:` for autocompleting emojis).
+2. Along with the prefix, type the initial letters of the text (e.g., `air` for `airplane`).
+3. Now hit the hotkey. You'd see suggestions being listed in the footer (if there are any).
+4. Cycle between different suggestions as described in above hotkey section. (Notice that a selected suggestion is colored in **black**)
+
+See the below GIF to better sync with above steps.
+
+![selected_footer_autocomplete](https://user-images.githubusercontent.com/55916430/116669526-53cfb880-a9bc-11eb-8073-11b220e6f15a.gif)
+
+### List of designated prefixes currently supported in ZT
+
+Now that you have a basic understanding of how autocompletes in ZT work, let's explore what all sorts of autocomplete prefixes ZT support, as of now. Note that the kinds of prefix supported also differ within each compose box. Thus, the prefixes listed below are grouped by type of compose box.
+
+You could refer to the above section to try out each of the below prefixes!
+
+#### Message compose autocomplete
+
+|Prefix Type|Prefix text(s)|Autocompleted text format|
+| :--- | :---: | :---: |
+|Mention a user|`@`, `@**`|`@**User name**`|
+|Mention a user group|`@*`|`@*User group*`|
+|Silently mention a user|`@_`, `@_**`|`@_**User name**`|
+|Get a stream|`#`, `#**`|`#**stream name**`|
+|Get an emoji|`:`|`:emoji_name:`|
+
+**NOTE:** Currently, the above autocompletes ignore a text containing whitespaces if it is specified as a prefix for autocompletion. (e.g., `#test he` wouldn't lead to `#**test here**`) See also [#925](https://github.com/zulip/zulip-terminal/issues/925).
+
+#### Other autocomplete fields
+
+
+All other message composing areas that support autocompletion don't require a prefix, i.e., the prefix for them is an empty string (`''`). Autocompletes here are contextual and are field-specific. See the below snapshots to better distinguish between various compose fields.
+
+|S. No.|Field|Autocompleted text format|
+| :---: | :--- | :---: |
+|1|Stream|`Stream name`|
+|2|Topic|`Topic name`|
+|3|PM recipient **\***|`Name <email>`|
+
+**\*** If a PM recipient's name contains comma(s) (`,`), they would be treated as comma-separated recipients.
+
+Stream message compose (for **1** and **2**):
```

Comment:
Since we're being specific, it might be better if we use specific terms as discussed on the stream, like "specifying stream and topic" and "specifying private message recipients". 

The other point I just noticed is, the cursor in your screenshots remains on the message content, could you perhaps put screenshots with the cursor on these fields? It might help with emphasizing the field we're describing here.